src/Cms/Core.php

```diff
@@ -242,6 +244,7 @@ public function fields(): array
 			'toggles'     => $this->root . '/fields/toggles.php',
 			'url'         => $this->root . '/fields/url.php',
 			'users'       => $this->root . '/fields/users.php',
+			'uuid'        => $this->root . '/fields/uuid.php',
```

Comment:
I think this might be a leftover from something because the file does not exist.

```suggestion
```